subcommand/server-acl-init/command.go

```diff
@@ -477,21 +550,8 @@ func (c *Command) bootstrapServers(logger hclog.Logger, bootTokenSecretName, sch
 // and then provides the token to the server.
 func (c *Command) setServerTokens(logger hclog.Logger, consulClient *api.Client,
 	serverPods []podAddr, bootstrapToken, scheme string) error {
-	// Create agent policy.
-	agentPolicy := api.ACLPolicy{
-		Name:        "agent-token",
```

Comment:
They share the same underlying rules. This allows different naming and description for client vs. server tokens in case their underlying rules end up changing in the future.